build/deploy/grafana.libsonnet

```diff
@@ -82,9 +74,8 @@ local base = import 'base.libsonnet';
 			},
 		},
 		service: base.Service(metadata, 'grafana') {
-			metadata: {
-				name: 'grafana',
-				namespace: metadata.namespace,
+			app:: 'grafana',
+			metadata+: {
 				annotations: {
 					'prometheus.io/scrape': 'true',
 					'prometheus.io/port': '3000',
```

Comment:
again, below if you set spec+: then you can remove selector, and the ports array is close to being set, if you want to override the base.Service object to account for this case